play-json/src/test/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializerSpec.scala

```diff
@@ -118,6 +118,41 @@ class PlayJsonSerializerSpec extends WordSpec with Matchers {
       }
     }
 
+    "pick up serializers for registry with migration" in withActorSystem(TestRegistry3) { system =>
+
+      val migratedEvent = MigratedEvent(addedField = 2, newName = "some value")
+
+      val serializeExt = SerializationExtension(system)
+      val serializer = serializeExt.findSerializerFor(migratedEvent).asInstanceOf[SerializerWithStringManifest]
+
+      val bytes = serializer.toBinary(migratedEvent)
+      val manifest = serializer.manifest(migratedEvent)
+
+
+      manifest shouldEqual classOf[MigratedEvent].getName + "#5"
+
```

Comment:
hi @MickJermsurawong , with this assert we now have 4 locations on the tests where we'd have to maintain the manifest format if it where to change. Also, this test is now coupled to the fact that `TestRegistry3` defines up to 5th generation of migrations.
What do you think we add a single test to assert the manifest format and use `serializer.manifest(migratedEvent)` everywhere else? (`everywhere else` == any test where a `string + "#num"` is used).